fbw-a32nx/src/wasm/fbw_a320/build.sh

```diff
@@ -13,7 +13,7 @@ if [ "$1" == "--debug" ]; then
   WASMLD_ARGS=""
   CLANG_ARGS="-g"
 else
-  WASMLD_ARGS="-O2 --lto-O2 --strip-debug"
+  WASMLD_ARGS="-O2 --lto-O2 --strip-debug --gc-sections"
```

Comment:
Suggest keeping this one in the common build flags so we strip unused cruft out of all binaries. Unfortunately GH won't let me suggest keeping the deleted line below (165).
```suggestion
  WASMLD_ARGS="-O2 --lto-O2 --strip-debug"
```